test/AngleTest.cpp

```diff
@@ -1,5 +1,6 @@
 #include <gtest/gtest.h>
-#include "roboteam_utils/Angle.h"
+#include <roboteam_utils/Angle.h>
```

Comment:
Can't this also be ```#include "Angle.h"``` as was done in ```Angle.cpp``` ?

Reply:
No, it's a different library so this is stylistically more correct. Angle.cpp and AngleTest.cpp are in different targets.